packages/gatsby-transformer-remark/README.md

```diff
@@ -63,3 +63,20 @@ A sample GraphQL query to get MarkdownRemark nodes:
   }
 }
 ```
+
+## Troubleshooting
+
+### Excerpts for non-latin languages
+
+By default, excerpts is avoid half-chopped words when truncating content.
```

Comment:
How about something like this:

> By default, `excerpt` uses `underscore.string/prune` which doesn't handle non-latin characters (https://github.com/epeli/underscore.string/issues/418). 
>
> If that is the case, you can set `truncate` option on `excerpt` field, like:

and your query after that?

Reply:
Thanks 👍, updated.